runtime/doc/eval.txt

```diff
@@ -3260,7 +3260,7 @@ declarations and assignments do not use a command.  |vim9-declaration|
 :let [{name}, ..., ; {lastname}] -= {expr1}
 :let [{name}, ..., ; {lastname}] .= {expr1}
 :let [{name}, ..., ; {lastname}] ..= {expr1}
```

Comment:
I'm not sure what these are even supposed to do. They don't seem used anywhere. `+=` works strange, while `-=`, `.=`, `..=` seem to always report "wrong variable type".